whylog_vim/input_reader/__init__.py

```diff
@@ -23,26 +21,32 @@ def parse_primary_key_groups(cls, content):
 
 
 class ConstraintReader(object):
-
-    CONSTRAINTS = {ConstraintType.IDENTICAL: IdenticalConstraint,}
-
     @classmethod
     def _parse_constraint(cls, lines):
```

Comment:
logic in this function is difficult to comprehend without an example. Please add example input/output in docstring.